BILLthebuilder/automart
# #166397769 Create user login endpoint

* It creates an endpoint for a user to signin
#### Description of Task to be completed?
* The `POST/api/v1/auth/signin` endpoint is created here
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ft-user-login-166397769` branch
* Clone the repo
* Run `npm install` to install package dependencies
* Run `npm test` to run tests
* All the tests should pass
* Run `npm start`
* Test the `/api/v1/auth/signin` request endpoint on postman
* It should return a `status` of `200`
* Errors are handled properly to prevent app crashing
#### What are the relevant pivotal tracker stories?
* [#166397769](https://www.pivotaltracker.com/story/show/166397769)
#### Screenshots
![Screenshot from 2019-06-19 06-20-35](https://user-images.githubusercontent.com/24655101/59734537-80600880-925a-11e9-90bf-0c89e69bfa4e.png)
